### PR TITLE
Fix import errors in some cases

### PR DESCRIPTION
**Pull request by Cereal2nd**
This fixes HA 0.90.1
this bumpes the version number

### CODE REVIEW

File: velbus/__init__.py [hunk 1]
```diff
@@ -155,4 +155,7 @@ def __str__(self):
 from velbus.module import Module
 from velbus.modules import *
 
-from velbus.controller import Controller, VelbusConnection
+from velbus.connection import VelbusConnection
+from velbus.connections import *
+
+from velbus.controller import Controller
```
[the remark this answers is not in the document]
Comment (Cereal2nd) [reply velbus/__init__.py]: This is the way python-velbus is written (we should solve this once)

File: velbus/__init__.py [hunk 2]
```diff
@@ -155,4 +155,7 @@ def __str__(self):
 from velbus.module import Module
 from velbus.modules import *
 
-from velbus.controller import Controller, VelbusConnection
+from velbus.connection import VelbusConnection
+from velbus.connections import *
```
[the remark this answers is not in the document]
Comment (Cereal2nd) [reply velbus/__init__.py]: This is the way python-velbus is written (we should solve this once)

File: velbus/__init__.py [hunk 3]
```diff
@@ -155,4 +155,7 @@ def __str__(self):
 from velbus.module import Module
 from velbus.modules import *
 
-from velbus.controller import Controller, VelbusConnection
+from velbus.connection import VelbusConnection
```
[the remark this answers is not in the document]
Comment (Cereal2nd) [reply velbus/__init__.py]: This is the way python-velbus is written (we should solve this once)

**Comment by Cereal2nd** [comment]
Bad PR will close

**Comment by Cereal2nd** [comment]
Bad PR do not merge !!!

**Comment by thomasdelaet** [comment]
Thanks. I did not merge but I have this problem with home assistant latest release. Will also look into it.

**Pull request closed without merging**